pkg/util/test/util.go

```diff
@@ -3155,7 +3155,7 @@ func validatePvcControllerPorts(cluster *corev1.StorageCluster, pvcControllerDep
 						for _, containerCommand := range container.Command {
 							if strings.Contains(containerCommand, "--secure-port") {
 								if len(pvcSecurePort) == 0 {
-									if isAKS(cluster) {
+									if isAKS(cluster) || IsK3sCluster() {
 										if strings.Split(containerCommand, "=")[1] != AksPVCControllerSecurePort {
```

Comment:
Now that it seems this is behavior not only for AKS, can we rename `AksPVCControllerSecurePort` to something like `defaultPvcControllerSecurePort` instead please

Reply:
Sure I will make the change